README.md

```diff
@@ -98,6 +98,9 @@ Update the var.tfvars file with values specific to your environment. Following i
  * `storage_type` : Storage provisioner to configure. Supported values: nfs (For now only nfs provisioner is supported, any other value won't setup a storageclass)
  * `volume_size` : If storage_type is nfs, a volume will be created with given size in GB and attached to bastion node. Eg: 1000 for 1TB disk.
  * `volume_storage_template` : Storage template name or ID for creating the volume. Empty value will use default template.
+ * `upgrade_image` : If set, OCP cluster will be upgraded to this image version.
+ * `pause_time` : Pauses playbook execution for a set amount of time in minutes.
```

Comment:
The description does not make sense as a user.
pause_time: Minutes to wait before starting to check the upgrade status once the upgrade command is run?
delay_time: Seconds to wait before re-checking the upgrade status subsequently?